onboarding/matplot.py

```diff
@@ -5,3 +5,25 @@
 - Be careful of data points that don't exist
 - Will require reading from CSV, can experiment with pandas
 '''
+from matplotlib import pyplot as plt
+from matplotlib import style
+#import numpy as np
+import pandas as pd
+from pyparsing import col 
```

Comment:
For these new dependencies, can you add them into the `requirements.txt` file so that they can be `pip install`ed easily?